src/azure-cli/azure/cli/command_modules/synapse/manual/_params.py

```diff
@@ -165,6 +165,9 @@ def load_arguments(self, _):
         # Component Version
         c.argument('spark_version', arg_group='Component Version', help='The supported Spark version is 2.4 now.')
 
+        # Spark config file
+        c.argument('spark_config_file_path', help='Spark pool properties configuration file.')
```

Comment:
>, [](http://example.com/codeflow?start=43&length=1)

Add it under group "Environment Configuration".
`arg_group='Environment Configuration'`